messaging/src/main/java/org/axonframework/commandhandling/gateway/AbstractRetryScheduler.java

```diff
@@ -130,12 +155,14 @@ public boolean scheduleRetry(CommandMessage commandMessage,
      * A builder class for the {@link RetryScheduler} implementations.
      * <p>
      * The default for {@code maxRetryCount} is set to a single retry.
+     * The default for {@code nonTransientFailurePredicate} is set to {@link DefaultNonTransientPredicate}.
```

Comment:
Nit: Other mentions of default settings are paired into a single sentence particularly.
So, something like this:

```suggestion
     * The default for {@code maxRetryCount} is set to a single retry and the {@code nonTransientFailurePredicate} defaults to {@link DefaultNonTransientPredicate}.
```

Reply:
By the way, if we rename the predicate, this bit of documentation should also change.